src/app/event/[id]/EventDetail.tsx

```diff
@@ -1,18 +1,103 @@
-import { getEvent } from '../../../api/events';
+'use client';
+
+import format from 'date-fns/format';
+import cx from 'classnames';
+import { useState } from 'react';
+
+import { HeartIcon } from '../../../components/icons/HeartIcon';
+import { mockEvent } from './mock';
+import { Event } from '../../../types/Event';
 
 type EventDetailProps = {
-  id: string;
+  event: Event;
+  className?: string;
 };
 
-export const EventDetail: React.FC<EventDetailProps> = async ({ id }) => {
-  const event = await getEvent(id);
+export const EventDetail: React.FC<EventDetailProps> = ({
+  event,
+  className = '',
+}) => {
+  const [isLiked, setIsLiked] = useState(false);
+
+  const handleLikeEvent = () => {
+    setIsLiked(!isLiked);
+  };
+
+  const { sport, location, price, date, level } = event;
 
   return (
-    <>
-      <h1 className="mt-24 px-20 text-center text-2xl font-medium leading-normal md:mt-14 md:px-0 lg:text-start lg:text-4xl">
-        {event?.name}
-      </h1>
-      <p className="mt-8">{event?.description}</p>
-    </>
+    <section
+      className={`${className} top-14 w-full max-w-lg rounded-md bg-white p-7 text-lg shadow-md lg:max-w-md`}
+    >
+      <article className="flex flex-col rounded-md bg-white">
+        <button
+          type="button"
+          onClick={handleLikeEvent}
+          aria-label={
+            isLiked ? 'Odstranit z oblíbených' : 'Přidat do oblíbených'
+          }
+          className="ml-auto"
+        >
+          <HeartIcon
+            className={cx(
+              'transition-transform duration-200 ease-in-out hover:scale-115 motion-reduce:hover:scale-100',
+              isLiked ? 'fill-primary' : 'fill-smoke-glass'
+            )}
+          />
+        </button>
+        <table className="text-s font-medium">
+          <tbody>
+            <tr className="border-b border-light-gray">
+              <td className="py-2 md:py-4">Cena</td>
+              <td className="py-2 font-light md:py-4">
+                {price === 0 ? 'ZDARMA' : `${price} Kč/osoba`}
```

Comment:
It might be the best to do `!!price ? 'Zdarma'...` in case the zdarma is undefined, you will go with the ZDARMA rather than `undefined Kc/osoba`

Reply:
It can not be undefined I beleive - .... default as "0" value.  But i like that solution!